src/test/java/ghostleg/domain/LadderHeightTest.java

```diff
@@ -0,0 +1,17 @@
+package ghostleg.domain;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class LadderHeightTest {
+    @Test
+    void atLeastOne() {
+        assertThrows(IllegalArgumentException.class, () -> new LadderHeight(0));
```

Comment:
```suggestion
        assertThrows(IllegalArgumentException.class, () -> new LadderHeight(LadderHeight.MIN_HEIGHT));
```
은 어떨까요? 이렇게하면 프로덕션 코드와 테스트코드를 연동해서 유지보수에 더 좋을거 같아요.